ui/src/app/shared/models.ts

```diff
@@ -170,6 +170,12 @@ export interface ApplicationDestination {
     name: string;
 }
 
+export interface ApplicationDestinationServiceAccounts {
```

Comment:
should it be in singular `ApplicationDestinationServiceAccount` 
 instead of `ApplicationDestinationServiceAccounts`? 
 just like `OrphanedResource` which is as an array `orphanedResources`

Reply:
Changed it to a singular name  `ApplicationDestinationServiceAccount`.